.github/CODEOWNERS

```diff
@@ -3,16 +3,20 @@
 * @CMSGov/easi
 
 # Frontend Changes
-/package-lock.json @CMSGov/easi-frontend
+/yarn.lock @CMSGov/easi-frontend
 /package.json @CMSGov/easi-frontend
 /src/ @CMSGov/easi-frontend
 /cypress/ @CMSGov/easi-frontend
 /public/ @CMSGov/easi-frontend
 /.storybook/ @CMSGov/easi-frontend
+*.js @CMSGov/easi-frontend
```

Comment:
I _think_ these wildcard patterns will only catch files with these extensions at the root of the repo. Is that your intention?

[Codeowners docs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax) say it follows .gitignore syntax, and the [Git docs for .gitignore](https://git-scm.com/docs/gitignore#_pattern_format) say "An asterisk "*" matches anything except a slash."; the `**/` pattern is needed to match in all directories.

Reply:
Good catch!

Reply:
@DylanSpOddball The only thing that I couldn't quite figure out was the example CODEOWNERS that GitHub shows has a lot of instances of things like `*.txt` for "owns all .txt files"

<img width="552" alt="image" src="https://user-images.githubusercontent.com/15203744/156405474-5d290247-f5d4-4230-b721-0667ffa534b0.png">

Reply:
@DylanSpOddball I created a PR that targeted this branch with a `.github/this_is_a_test.tsx` file, and it caught it with part of this pattern

I think this means we should be good with single-asterisk patterns, though it's odd because you'd _think_ that'd go against .gitignore structure.

![image](https://user-images.githubusercontent.com/15203744/156406046-4426feb6-a3c1-4ee5-b40d-2cf3e3d627d9.png)

Reply:
Hmm, ok. Good to know, thanks for testing.